src/controllers/dishController.js

```diff
@@ -18,10 +23,9 @@ exports.createDish = async (req, res) => {
     const savedDish = await newDish.save();
 
     // Send the saved dish as a response
-    res.status(201).json(savedDish);
+    res.status(201).json({ savedDish });
   } catch (error) {
-    // Handle server error
-    res.status(500).json({ error: 'Internal server error' });
+    res.status(500).json(error.message);
   }
```

Comment:
Inside the createDish, all you need to do is to put the Id of the chef inside the params object that inside the req object.

So for example when you do a request from postman to create a dish, what you need to do is just to put the id of the chef that you want the dish to be related to manually, so when the dish get create it will has the id of a chef that you add his Id manually.